.misc/appveyor.yml

```diff
@@ -32,32 +41,90 @@ branches:
+
 install:
-  # Prepend newly installed Python to the PATH of this build (this cannot be
-  # done from inside the powershell script as it would require to restart
-  # the parent CMD process).
-  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
-  - "SET PATH=%PATH%;C:\\Program\ Files\\Git\\mingw64\\bin"
+  # Show initial state
+  - powershell -c "$PSVersionTable"
+  # Uncomment to debug
+  # printenv
+  - python --version
+  - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
+  - python -m pip --version
+  - python -c "import setuptools; print(setuptools.__version__)"
+  - node --version
+  - which npm
+  - npm --version
+  - npm config get prefix
+  - which gcc & exit 0
+  - gcc --version & exit 0
+  - which cl & exit 0
+
+  # Stores environment in registry, with minor tweaks
+  - python .misc/store_env_in_registry.py
+  - refreshenv
+
+  # Set up AppVeyor product versions, and install dummy choco entries for them
+  - ps: . .misc/FudgeCI.ps1; Initialize-AppVeyorVM
+  - refreshenv
+  - echo %PATH%
+  # Avoid tools finding and using MinGW
+  - mv C:\MinGW %TEMP%
+  # TODO: Avoid tools finding and using Visual Studio
+
+  # Show updated SOE; versions should be as defined in top of the Fudgefile
+  - python --version
+  - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
+  - node --version
+  - which npm
+  - npm --version
+  - npm config get prefix
+  - which gcc & exit 0
+  - gcc --version & exit 0
+
+  - "%MSYS_BIN%\\date.exe"
+  # Install remainer of the Fudgefile with chocolatey using Fudge
+  - ps: . .misc/Fudge.ps1 install
+  - refreshenv
+  - echo %PATH%
+
 
   # Check that we have the expected version and architecture for Python
   - "python --version"
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
-  - "%CMD_IN_ENV% python -m pip install --upgrade setuptools==21 pip==9.0.1"
-  - "%CMD_IN_ENV% python -m pip install -r test-requirements.txt \
-    -r requirements.txt -r docs-requirements.txt"
+  # Confirm other versions
+  - node --version
+  - which npm
+  - npm --version
+  - npm config get prefix
+  # Newly installed versions
+
+
+  - "%MSYS_BIN%\\date.exe"
 
 build: false  # Not a C# project, build stuff at the test step instead.
 
 test_script:
-  - "%CMD_IN_ENV% py.test"
-  - "%CMD_IN_ENV% python setup.py install"
-  - "%CMD_IN_ENV% python -m pip install \
-    git+https://github.com/coala/coala-bears"
+  - python -m pip --version
+  - python -c "import setuptools; print(setuptools.__version__)"
+
+  - py.test
+
+  - python setup.py install
+
+  - python -m pip install
+      git+https://github.com/coala/coala-bears#egg=coala-bears
   - npm install csslint -g
   # https://github.com/coala/coala/issues/5619
   - rm docs/Developers/coala_settings.rst
-  - sed -i '/ShellCheckBear/d' .coafile
-  - "%CMD_IN_ENV% coala --ci"
+  # https://github.com/coala/coala/issues/5985 and
+  # https://github.com/coala/coala/issues/6028 and others:
+  - rm coalib/misc/Asyncio.py
```

Comment:
Oh, I thought you had a different hacky fix :P

Reply:
Nope - a hack until we get a real solution, or drop py34 (but py34 is helpful for the bears travis trusty (and precise?) builds, which are needed for some of the languages)